dm/ctl/master/shard_ddl_lock.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// NewShardDDLLockCmd creates a ShardDDLLock command.
+func NewShardDDLLockCmd() *cobra.Command {
```

Comment:
Not sure if we can merge `shard-ddl-lock` and `show-ddl-lock` into one command

Reply:
Any suggestion?

Reply:
Maybe shard-ddl-lock show?